examples/GizmoTest/GizmoTest.ino

```diff
@@ -16,7 +16,16 @@
 #define EPD_RESET   PIN_A3 // can set to -1 and share with microcontroller Reset!
 #define EPD_BUSY    -1 // can set to -1 to not use a pin (will wait a fixed delay)
 
-Adafruit_IL0373 display(152, 152, EPD_DC, EPD_RESET, EPD_CS, SRAM_CS, EPD_BUSY);
+// You will need to use Adafruit's CircuitPlayground Express Board Definition
+// for Gizmos rather than the Arduino version since there are additional SPI
+// ports exposed.
+#if (SPI_INTERFACES_COUNT == 1)
+  SPIClass* spi = &SPI;
+#else
+  SPIClass* spi = &SPI1;
+#endif
```

Comment:
Could this magic be put in EPD.h so it all happens via the base class?

Reply:
That would probably limit the library severely since it's meant to also run on breakouts and a number of microcontrollers.

Reply:
Ah. OK. Then do the other classes need a similar update so an alternate SPI bus can be specified for them as well?

Reply:
Yeah, that would probably be a good idea, but I kind of wanted to do this as a test case first.